### PR TITLE
#2: Firebase realtime database integration

### PR DESCRIPTION
**Pull request by isotopeee**
- Integrate firebase realtime database

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,38 +1,54 @@
...
     if (status !== 'granted') {
       this.setState({
         errorMessage: 'Permission to access location was denied',
       });
     }
+  };
 
-    let location = await Location.getCurrentPositionAsync({});
-    this.setState({ location });
+  _updateLocation = () => {
```
Comment (jmaicaaan) [inline App.js]: I think having the _ is little not neat as it not provides any private or public methods.

Comment (jmaicaaan) [reply App.js]: The naming for the `_updateLocation` is unclear. It is doing a lot of things and not directly based from the method definition.

Suggestions: 
- Create a functional method that will transmit the location
- Create a method that will trigger the position watching

File: firebase/firebase.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import firebase from 'firebase/app';
+import 'firebase/database';
+
+const config = {
```
Comment (jmaicaaan) [inline firebase/firebase.js:4]: I believe for now this is perfectly fine, but moving forward, having a config outside of a codebase is much better as it does not interfere with codebase business logic. And also, can cater different environments.

**Review by jmaicaaan — COMMENTED** [review]
The looks good, though I haven't tested or run it through the simulator.

**Comment by isotopeee** [comment]
Closing in favor of #5

**Pull request closed without merging**